Rocket.Chat/Controllers/Base/BaseViewController.swift

```diff
@@ -25,5 +19,16 @@ class BaseViewController: UIViewController {
             action: nil
         )
     }
+}
 
+protocol Alerter {
+    func alert(title: String, message: String, handler: ((UIAlertAction) -> Void)?)
+}
+
+extension Alerter where Self: UIViewController {
+    func alert(title: String, message: String, handler: ((UIAlertAction) -> Void)? = nil) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "OK", style: .default, handler: handler))
+        present(alert, animated: true, completion: nil)
+    }
```

Comment:
Let's remove this code from here and create it as an extension of a separated protocol? Because it can be used in other UIViewControllers.